data/xml/2009.mtsummit.xml

```diff
@@ -0,0 +1,977 @@
+      <author><first>Susan</first><last>Converse</last></author>
+      <author><first>Rachel</first><last>Richardson</last></author>
+      <author><first>John S.</first><last>White</last></author>
+      <author><first>Tucker</first><last>Maney</last></author>
+      <url hash="7f05bd43">2009.mtsummit-government.8</url>
+      <bibkey>van-ess-dykema-etal-2009-translation</bibkey>
+    </paper>
+  </volume>
+  <volume id="tutorials" ingest-date="2021-08-02">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit XII: Tutorials</booktitle>
+      <address>Ottawa, Canada</address>
+      <month>August 26-30</month>
+      <year>2009</year>
+    </meta>
+    <paper id="1">
+      <title>Introduction to Machine Translation</title>
+      <author><first>Mike</first><last>Dillinger</last></author>
+      <attachment type="presentation" hash="527b81a8">2009.mtsummit-tutorials.1.Presentation.pdf</attachment>
+      <bibkey>dillinger-2009-introduction-machine</bibkey>
+    </paper>
+    <paper id="2">
+      <title>Machine Learning Approaches for Dealing with Bilingual Data in Statistical Machine Translation</title>
+      <author><first>Gholamreza</first><last>Haffari</last></author>
+      <bibkey>haffari-2009-machine</bibkey>
+    </paper>
+    <paper id="3">
+      <title>Tools for Translation: Current Practices, State of the Art and Capability Gaps</title>
+      <author><first>Jennifer</first><last>DeCamp</last></author>
+      <url hash="4a4ec5f2">2009.mtsummit-tutorials.3</url>
+      <bibkey>decamp-2009-tools</bibkey>
+    </paper>
+    <paper id="4">
+      <title>The Business Case for Machine Translation</title>
+      <author><first>Donald A.</first><last>DePalma</last></author>
+      <attachment type="presentation" hash="7c613d76">2009.mtsummit-tutorials.4.Presentation.pdf</attachment>
+      <bibkey>depalma-2009-business</bibkey>
+    </paper>
+    <paper id="5">
+      <title>Postediting Machine Translation Output Guidelines</title>
+      <author><first>Sharon</first><last>O’Brien</last></author>
+      <author><first>Johann</first><last>Roturier</last></author>
+      <author><first>Giselle</first><last>de Almeida</last></author>
+      <attachment type="presentation" hash="fa10d7dd">2009.mtsummit-tutorials.5.Presentation.pdf</attachment>
+      <bibkey>obrien-etal-2009-postediting</bibkey>
+    </paper>
+    <paper id="6">
+      <title>Up close and personal with a Translator – How Translators Really Work</title>
+      <author><first>Alain</first><last>Desilets</last></author>
+      <author><first/><last>UQO</last></author>
+      <author><first>Geneviève</first><last>Patenaude</last></author>
+      <url hash="8a028bf1">2009.mtsummit-tutorials.6</url>
+      <bibkey>desilets-etal-2009-close</bibkey>
+    </paper>
+  </volume>
+  <volume id="caasl" ingest-date="2021-08-02">
+    <meta>
+      <booktitle>Workshop on Computational Approaches to Arabic-Script-based Languages</booktitle>
```

Comment:
"Proceedings of the Third Workshop on Computational Approaches to Arabic-Script-based Languages (CAASL3)"